libs/sdk-ui-dashboard/src/presentation/widget/common/configuration/hooks/useAttributeFilterConfiguration.ts

```diff
@@ -0,0 +1,47 @@
+// (C) 2023 GoodData Corporation
+import { useEffect, useMemo } from "react";
+import {
+    queryAttributeByDisplayForm,
+    QueryAttributeByDisplayForm,
+    useDashboardQueryProcessing,
+} from "./../../../../../model";
+
+import { IAttributeMetadataObject, IDashboardAttributeFilter } from "@gooddata/sdk-model";
+
+interface IUseAttributeFilterConfigurationReturnType {
+    attributes: IAttributeMetadataObject[] | undefined;
+    attributesLoading: boolean;
+}
+
+export const useAttributeFilterConfiguration = (
```

Comment:
Why do we need this? How is it different than `useAttributes`?

Reply:
Moved to shared folder _staging, so it can be reused.